### PR TITLE
OKTA-188842: Adds event type catalog data and templated page

### PR DESCRIPTION
**Pull request by bobtiernay-okta**
## Description:

Adds event type catalog data and templated page

- Adds new event-types.md page
- Links from system_log.md page
- Adds support for removing ToC via page variables

### Resolves:

* [OKTA-188842](https://oktainc.atlassian.net/browse/OKTA-188842)

### CODE REVIEW

File: _source/_docs/api/resources/system_log.md [hunk 1]
```diff
@@ -476,9 +476,8 @@ Describes an IP address used in a request.
 
 Event types categorize event instances by action and recorded in a LogEvent's [`eventType`](#attributes) attribute. They are key to navigating the system log via [Expression Filters](#expression-filter).
 
-The following sections outline the major event types captured by the system log.
+The following sections outline the key event types captured by the system log. For a complete listing see the [Event Types catalog](/docs/api/resources/event-types#catalog).
```
Comment (bobtiernay-okta) [inline _source/_docs/api/resources/system_log.md]: Added link.

File: _source/_docs/api/resources/event-types.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+# To update the data in this table, please see https://bit.ly/2xsgk47
+ 
+layout: docs_page
+weight: 2
+title: Event Types
+toc: false
```
Comment (bobtiernay-okta) [inline _source/_docs/api/resources/event-types.md]: Now can disable ToC here.

File: _source/_data/docs.yml [hunk 3]
```diff
@@ -30,6 +30,7 @@
     - apps
     - authorization-servers
     - events
+    - event-types
```
[the remark this answers is not in the document]
Comment (jakubvul) [reply _source/_data/docs.yml]: My sneaking suspicion is: nothing

**Comment by jakubvul** [comment]
@nbarbettini @bobtiernay-okta is there some reason we're not putting this page alongside the other reference pages in the sidebar?

**Comment by bobtiernay-okta** [comment]
@jakubvul I think one reason might be cause it's not technically an API and the others appear to have that in common. But I'm good to add it. Let me know.

**Pull request merged**